analyzers/tests/SonarAnalyzer.UnitTest/Rules/FileShouldEndWithEmptyNewLineTest.cs

```diff
@@ -29,12 +29,11 @@ public class FileShouldEndWithEmptyNewLineTest
     {
         [TestMethod]
         public void FileShouldEndWithEmptyNewLine() =>
-            OldVerifier.VerifyAnalyzer(new[]
-                {
-                    @"TestCases\FileShouldEndWithEmptyNewLine_EmptyLine.cs",
-                    @"TestCases\FileShouldEndWithEmptyNewLine_NoEmptyLine.cs",
-                    @"TestCases\FileShouldEndWithEmptyNewLine_EmptyFile.cs"
-                },
-                new FileShouldEndWithEmptyNewLine());
+            new VerifierBuilder<FileShouldEndWithEmptyNewLine>().AddPaths(
+                    "FileShouldEndWithEmptyNewLine_EmptyLine.cs",
+                    "FileShouldEndWithEmptyNewLine_NoEmptyLine.cs",
+                    "FileShouldEndWithEmptyNewLine_EmptyFile.cs")
+                .WithAutogenerateConcurrentFiles(false)
```

Comment:
As you enable them by default, consider: WithoutAutogenerateConcurrentFiles()

Reply:
I've considered this on the `WithConcurrentAnalysis` and it would be a bit irregular and save only 2 characters.

Reply:
It is more about explaining that you are deviating from the default configuration.